importer/make_KMB_info.py

```diff
@@ -473,9 +473,30 @@ def generate_content_cats(self, item):
             item.make_item_class_categories(self.category_cache)
             item.make_item_keyword_categories(self.category_cache)
 
-        # Add parish/municipality categorisation when needed
-        if item.needs_place_cat:
-            item.make_place_category()
+            # Add parish/municipality categorisation when needed
```

Comment:
You don't want to put the municipality/parish categorisation under the `if not found_commonscat`.

I would suggest simply putting the `item.exact_category_from_name` under its own `if not found_commonscat` but after the parish/municipality categorisation has been dealt with.

Reply:
Moved it to a separate function.